compiler/tests-spec/testData/diagnostics/notLinked/dfa/pos/25.fir.kt

```diff
@@ -52,8 +52,8 @@ inline fun <reified T : CharSequence>case_4(x: Any?) {
     (x as? T)!!
     if (x is T?) {
         <!DEBUG_INFO_EXPRESSION_TYPE("T? & kotlin.Any?")!>x<!>
-        <!DEBUG_INFO_EXPRESSION_TYPE("T? & kotlin.Any?")!>x<!>.<!INAPPLICABLE_CANDIDATE!>length<!>
-        <!DEBUG_INFO_EXPRESSION_TYPE("T? & kotlin.Any?")!>x<!>.<!INAPPLICABLE_CANDIDATE!>get<!>(0)
+        <!DEBUG_INFO_EXPRESSION_TYPE("T? & kotlin.Any?")!>x<!>.<!UNSAFE_CALL!>length<!>
```

Comment:
This (25) and below two (26, 27) share the same issue. Filed https://youtrack.jetbrains.com/issue/KT-44559